### DISCUSSION TITLE
Wich what of these models i can use LORA's❓

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1412] [Q&A]
![Screenshot_2](https://github.com/user-attachments/assets/96186606-df7a-4dd5-b19b-f1f9f565371a)

And also some creators give the indication of use **Diffusion in Low Bits**: with "Automatic (fp16 LoRA)".

Thx in advance.

**Comment by rlewisfr** [comment]
Just my non-expert opinion, but I have found that Diffusion Low Bits automatic (vanilla) is not really automatic anything.  If you are using LORA's, the Automatic fp16 LORA seems to work most of the time, but the bnb-nf4 (fp16 Lora) is really the only thing that I can get LORAs to work for nf4.  Even then, LORA strength is really off (at least for nf4).  1.0 is really 0.0, and most LORAs don't have much effect until 1.5 and I run most at 2.0.  Then things get really weird when using sub 1.0 strength, with 0.5 producing strange effects indeed.

Again, my non-expert observations.
